### ISSUE TITLE
Menu - update drilldown inline filter example markup

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#9777]
**Describe the enhancement or change**
Currently the [Menu with drilldown and inline filter](https://www.patternfly.org/components/menus/menu/#with-drilldown-and-inline-filter) example is rendering the search filter (drilled into level 2) as a div inside a `ul` element, causing an axe error to be flagged (since only `li` element should be direct children). Either the search input should be moved outside of the menu list or should be rendered within an `li` as well.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

**Comment by mcoker** [comment]
> should be rendered within an `li` as well.

If we want to keep it in the list like that, one option could be to create a generic `li` that does not have any sort of hover or other treatment and could be used for putting a non-interactive type of thing in. Then the `<MenuSearch>` (or whatever) could be placed inside of it.

**Comment by thatblindgeye** [comment]
Let's try updating the wrapper for the filter to an li and see if that resolves it short term. We can maybe re-evaluate this implementation on a deeper level in the future.
